data-collection/src/main/java/refactoringml/db/Project.java

```diff
@@ -26,8 +24,14 @@
 
 	private int commits;
 
-	//Collect instances of non-refactorings with different Ks e.g, 25,50,100 commits on a file without refactorings
+	//Collect instances of non-refactorings with different Ks e.g, 25,50,100 commits on a file without refactorings, this is for hibernate
 	private String commitCountThresholds;
+	//Collect instances of non-refactorings with different Ks e.g, 25,50,100 commits on a file without refactorings
+	@Transient
+	private List<Integer> commitCountThresholdsInt;
```

Comment:
Why is this here? The list of Ks doesn't seem to be a characteristic of `Project` itself. I understand you tried to encapsulate this list of ints somewhere, otherwise you'd have to pass it all over, but maybe that's more semantic/clear than hiding it here.